sys-auth/libfprint/libfprint-1.90.7.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson udev
+
+DESCRIPTION="Library to add support for consumer fingerprint readers"
+HOMEPAGE="https://cgit.freedesktop.org/libfprint/libfprint/ https://github.com/freedesktop/libfprint https://gitlab.freedesktop.org/libfprint/libfprint"
+SRC_URI="https://github.com/freedesktop/libfprint/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="examples +introspection gtk-doc"
+
+RDEPEND="
+	dev-libs/glib:2
+	>=dev-libs/libgusb-0.3.1
+	dev-libs/nss
+	!>=sys-auth/libfprint-1.90:0
```

Comment:
Well, since we use lexicographical sorting, this'd be either at the top or in the bottom. We usually separate blockers to bottom.

Reply:
Fixed